flink-docs/README.md

```diff
@@ -28,7 +28,7 @@ The `RestAPIDocGenerator` can be used to generate a full reference of the REST A
 To integrate a new endpoint into the generator
 1. Add a new `DocumentingRestEndpoint` class to `RestAPIDocGenerator` that extends the new endpoint class
 2. Add another call to `createHtmlFile` in `RestAPIDocGenerator#main`
-3. Regenerate the documentation by running `mvn package -Dgenerate-rest-docs -pl flink-docs -am -nsu`
+3. Regenerate the documentation by running `mvn package -Dgenerate-rest-docs -pl flink-docs -am -nsu -DskipTests`
```

Comment:
I intentionally left this option out. This guarantees the correctness of the output since it both tests the generator before the generation, and the completeness afterwards. There's a lot of additional options one _could_ add, but I'd prefer if we stick to the _necessary_ ones.

Reply:
ah wait a second, this would run the test for all modules. urgh...